plugins/tor/tor_

```diff
@@ -88,6 +138,18 @@ class TorPlugin(object):
     @staticmethod
     def autoconf():
         try:
+            import stem
+
+        except ImportError as e:
+            print('no ({})'.format(e))
```

Comment:
A manually written error message is probably easier to understand for non-pythonic people.
Maybe the following?
```python
print('no (failed to import the required python module "stem": {})'.format(e))
```
```